web-app/js/portal/cart/DownloadEstimator.js

```diff
@@ -119,5 +128,29 @@ Portal.cart.DownloadEstimator = Ext.extend(Object, {
         var fileSizeImage = OpenLayers.i18n("faError");
 
         return String.format(html, downloadTimeoutMessage, fileSizeImage);
+    },
+
+    _spatialSubsetIntersect: function(collection) {
+
+        var intersect = true;
+        var filters = collection.getFilters();
+        var params = filters.filter(function(filter) {
+            return filter.isNcwmsParams;
+        })[0];
+        var bounds = collection.getBounds();
```

Comment:
As the bounds is an Openlayers bounds you can use Openlayers functions to determine if 'rectangles intercect'
http://dev.openlayers.org/docs/files/OpenLayers/BaseTypes/Bounds-js.html#OpenLayers.Bounds.containsBounds